model/event/event_finder.go

```diff
@@ -39,6 +39,20 @@ func Find(coll string, query db.Q) ([]EventLogEntry, error) {
 	return events, nil
 }
 
+func FindPaginated(coll string, query db.Q, skip int) ([]EventLogEntry, error) {
```

Comment:
Instead of adding this function, I would either add the skip as a param to the original MostRecentHostEvents query or have this be a wrapper around that. I don't believe it makes much sense to have a skip without a limit, so we should try to pass those parameters together when possible

Reply:
done. thanks!